software/digistorm.yml

```
@@ -0,0 +1,11 @@
+name: "Digistorm"
+website_url: "https://digistorm.app/"
+source_code_url: "https://codeberg.org/ladigitale/digistorm"
+description: "Create collaborative surveys, quizzes, brainstorms, and word clouds (documentation in French)."
+licenses:
+  - AGPL-3.0
+platforms:
+  - Nodejs
+tags:
+  - Groupware
```

```suggestion
  - Groupware
  - Polls and Events
```

The survey feature is similar to those found at https://awesome-selfhosted.net/tags/polls-and-events.html

---

Dubly noted, i will also look at adding multiple tags within the future. Committing that suggestion now